stream_alert/alert_processor/main.py

```diff
@@ -79,6 +81,22 @@ def run(alert, region, function_name, config):
         LOGGER.error('Invalid alert format:\n%s', json.dumps(alert, indent=2))
         return
 
+    # Load enrichments
+    modules_to_import = set()
+    # walk the enrichments directory to dymanically import
+    for root, _, files in os.walk('enrichments'):
+        filtered_files = [enrichment_file for enrichment_file in files if not (
+            enrichment_file.startswith(('.', '__init__')) or
```

Comment:
This is an odd indentation - it should be aligned with `[enrichment_file... ` in the preceding line.

Tip: The `yapf --in-place <path/to/file.py>` will help with file formatting.